assets/src/edit-story/karma/_init.js

```diff
@@ -14,14 +14,32 @@
  * limitations under the License.
  */
 
+/* global karmaPuppeteer */
```

Comment:
Ideally declared as a (readonly) global in ESLint config

Reply:
Done